facia-press/app/frontpress/PressedCollectionDeduplication.scala

```diff
@@ -51,7 +57,10 @@ object PressedCollectionDeduplication {
       collectionV: PressedCollectionVisibility,
   ): PressedCollectionVisibility = {
     // Essentially deduplicate the backfill of collectionV using header values values from accum's elements curated and backfill
-    val accumulatedHeaderURLsForDeduplication: Seq[String] = getHeaderURLsFromCuratedAndBackfilledAtDepth(accum, 10)
+    val accumulatedHeaderURLsForDeduplication: Seq[String] =
```

Comment:
This change makes sense to me but it took me a while to understand the flow. Would it make sense to rename this variable to `headerURLsForDeduplication` as this sequence is no longer constructed only from the accumulator but also from the curated content of the `collectionV`? 

(btw I think I would also rename `collectionV` to just `collection`)

Reply:
Thanks for this! I've updated the name of the `headersURLs...` variable as you suggested. 

I can see the case for renaming `collectionV`, but it seems like it was quite a conscious choice to highlight that it's a `pressedCollectionVisibility` (also in the `pCVs` variable) and I'm not 100% sure I understand the implications of that,  so I'm inclined to leave it for now  until we have a clearer understanding of this bit of functionality?

Reply:
Yeah that's perfectly fine. Like Olly said there is an existing complexity in how deduplication works but I think your work was a really good opportunity for us to think / understand a bit better about some parts of this logic.